src/screens/WalletScreen.js

```diff
@@ -13,8 +13,16 @@ import AssetRow from '../components/AssetRow';
 import { accountGetAssets } from '../redux/_account';
 
 class WalletScreen extends Component {
+  state={ refreshing: false }
```

Comment:
Don't mix React state with Redux state, just handle all state in Redux

Reply:
cc @Dobrokhvalov

Reply:
@pedrouid done